webplugin/js/app/km-message-markup-1.0.js

```
@@ -28,17 +30,13 @@ Kommunicate.messageTemplate = {
         switch (true) {
             case data.fileMeta.contentType.indexOf('image') != -1:
                 data.attachmentDownloadClass = 'vis';
-                data.attachmentClass += " km-img-template-container";
-                return Mustache.to_html(
-                    Kommunicate.messageTemplate.getAttachmentTemplate(),
-                    data
-                );
+                data.attachmentClass += ' km-img-template-container';
+                return Mustache.to_html(Kommunicate.messageTemplate.getAttachmentTemplate(), data);
                 break;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)